src/hooks/tough-cookie.ts

```diff
@@ -0,0 +1,17 @@
+import { CookieJar } from 'tough-cookie';
+import { Options } from 'got-cjs';
+
+interface CookieData {
+    toughCookie?: CookieJar;
```

Comment:
So is it a `toughCookie`: "cookie" or a `cookieJar`? Maybe `toughCookieJar`?

Reply:
Let's name it just `cookieJar`